content/Contributing to open source: An overview of the django-drip-campaings project - Bruno Michetti.md

```diff
@@ -0,0 +1,217 @@
+Pypi (that stands for Python Package Index) is a repository of software for the [Python](https://www.python.org/) programming language that helps us to find and install software developed by the Python community. One of our contributions to the OSS community is to publish and maintain the named project and I want to share that experience in this blog.
+
+### The Django Drip Campaigns Project
+
+Django Drip Campaigns Project is an open source app to create drip campaigns for email using [Django’s admin](https://docs.djangoproject.com/en/3.1/ref/contrib/admin/) interface and a User queryset. This means that using this app, you are able to easily schedule and send emails to the registered users, creating querysets in a friendly interface.  
+For example, you can very quickly configure your app to send a periodic email to the users that haven't logged in in a day or more. Or you can send a custom email to the users that are registered more than a week ago, etc. You only need to define one or more querysets in the admin page.
+If you want to install it and work with it, you can follow the simple steps documented in [the github page of Django Drip Campaigns](https://github.com/rootstrap/django-drip-campaigns).  
+We didn't create this project, we just made a fork of the one written by [Zapier](https://zapier.com/z/qO/) that is no longer maintained. After this we worked on fixing some bugs, doing a couple of enhancements, creating more documentation and publishing the app as a pypi package.  
+Since it's published, any programmer that wants to use this project in his django app is able to quickly download it with a simple command described in the mentioned docs.
+
+### Features
+
+After a successful installation and configuration, you are able to easily create drip campaigns for email using the admin interface of Django. If you don't have an admin user, create one with the [django createsuperuser command](https://docs.djangoproject.com/en/3.0/intro/tutorial02/#creating-an-admin-user). After this, login into Django's admin page and click on `Drips` to manage them. There you are able to:
+
+- View created and sent drips.
+- Create a new drip.
+- Select and delete drips.
+
+#### Creating a new drip
+
+Click on the `ADD DRIP +` button to create a new drip. You will see a page like this:
+
+<img src="images/add_drip_page.png" alt="Create Drip" width="700" />
+
+In the next section I will talk about the queryset rules that you can build in an easy way to create very useful drips.
+
+#### Queryset rules
+
+When you create a drip, you need at least one queryset so the users in it will be the ones that receive the defined email. In the `METHOD TYPE` menu you are able to choose to filter or to exclude the users in the defined queryset.  
+On the other hand, when you click in the `FIELD NAME OF THE USER` input, you will see the fields of your User model, and the fields of your user model in the models related to it:
```

Comment:
```suggestion
On the other hand, when you click in the `FIELD NAME OF USER` input, you will see the fields of your User model, and the fields of your user model in the models related to it:
```
Just changing this because the photo shows the column without `THE` word.

Reply:
`FIELD NAME OF THE USER` is the name in the django admin. I have to keep this name

Reply:
👌 As I said, I just commented on this because the photo shows without the `THE` word but no problem leaving it as it is 😄

Reply:
You are right! my bad